src/js/components/DataTable/doc.js

```diff
@@ -378,12 +378,26 @@ export const themeDoc = {
     }`,
     defaultValue: undefined,
   },
-
   'dataTable.header.pad': {
     description: 'The pad around the contents of the header cell.',
     type: 'string | object',
     defaultValue: undefined,
   },
+  'dataTable.resize.hover.color': {
+    description: 'The color of the resizer when hovered over.',
+    type: 'string | object',
+    defaultValue: undefined,
+  },
+  'dataTable.resize.hover.side': {
```

Comment:
I'm trying to think of a case where a theme would want a different value for this. If we don't think it will be used. I think we should remove it.

Reply:
Right, since this is wrapping around an empty box with padding, putting any side except for `end` is just wrapping around something somewhat invisible. The default resize theme option sets `end` as default, but it can still be overridden. However, I don't think there is any loss from removing it here. Should I remove it only from the docs or from the theme structure as well?

Reply:
Hmm... my inclination is to automatically apply it if `resizer.hover.border` exists in the theme. On the other hand, I could see just being more explicit about this driving a Box border and that the theme probably wants this to be 'end'.

Reply:
Gotcha -- it feels like to be consistent with default resize, we should set this to `end` by default and then also be explicit in the docs that this drives a box border but that `end` is the recommended value.

Reply:
Are you thinking we have the default resize hover be `resize.hover.border = { side: 'end', color: 'text-strong' }`? It would seem odd to have only `side`, which wouldn't really do anything.

Reply:
I guess it's a question of if we want to introduce a new default visual change for existing grommet users. The reason I was just thinking of defining `end` (or `end` and color: `border`) was so that there were no new visual behaviors on grommet by default. 

However, if we are not introducing any visual changes on grommet base.js, then we could just leave the values undefined in the theme and do:
1. add a check in the component where if `color` or `size` is specified for border, but side isn't that `side=end` gets applied automatically. 
2. recommend in the docs that `end` is used for this value

Reply:
I lean towards #1

Reply:
gotcha, I would agree. Let me implement that.